### ISSUE TITLE
Listing active members of a team chat?

### ISSUE THREAD

**Issue by ViperGeek** [keybase/keybase-issues#3618]
We just moved our small team to Keybase, replacing a decade-old Openfire XMPP server.  Keybase team chat is superior in all possible ways to the old server, but one thing I didn't think I'd miss is the ability to find out who's here/up/awake/active in the team chat window.

I see that /flip somehow can determine active clients in the chat room (e.g. "Secured by\n 2 participants").  Is there any way a member of a team, active in a chat room, can see who else is active?

**Comment by RayOei** [comment]
Yes, for team use that would be very useful. I can imagine something comparable with Slack, for instance, where, as a user you can set it (or hide yourself). As a company user I think Keybase is actually (or at least can be) a competitor in that space as well ;-)
